### ISSUE TITLE
Missing Features: Accessibility - Select Sections menu 

### ISSUE THREAD

**Issue by Sharaf-Mansour** [Sharaf-Mansour/CVBuilder#36]
Self Explained 
![image](https://user-images.githubusercontent.com/55330747/147332852-79a5d622-43aa-496f-895a-4bc057ddebcd.png)
Same idea as that
